### ISSUE TITLE
Scrum Updates 1

### ISSUE THREAD

**Issue by JimmyRowland** [JimmyRowland/team_sporty#1]

**Comment by WilliamJizh** [comment]
1. I contributed to the project brainstorm and feature discussion, as well as a part of the prototype design.
2. Being unfamiliar with web development, I had to gather more knowledge about it along the way to purpose more grounded and achievable ideas.
3. I'll try to contribute to the component building in the upcoming iteration.

v2u1b

**Comment by gparappilly** [comment]
1. I helped contribute to the discussion/brainstorm of our app and features that we could implement.  I set up the google drive for this project, and did a lot of the writing for the ReadMe file. 
2. A challenge that I ran into was the fact that we're not gonna be able to meet in person. So at times, it hard to explain what you're thinking or want to do.
3. For the next iteration, I plan to be familiar with React and hopefully start contribute to the basic template of our web app. As well, I plain to hopefully refine some of our ideas so we know exactly what we need to do for this project.

n1u9a

**Comment by engineereng** [comment]
1. I contributed 3 ideas to the project brainstorm, and acted as note-taker and call host whenever we discussed our project.
2. I found it difficult to visualize the scope of our ideas and the amount of work that would be necessary to implement these tasks.
3. For the next iteration, I will practice using React so I can make one of the components. I plan to talk with the group to figure out a timeline for the project and discuss our roles.

i5m1b
